types/react/index.d.ts

```diff
@@ -1656,16 +1657,16 @@ declare namespace React {
         // Standard HTML Attributes
         accessKey?: string;
         className?: string;
-        contentEditable?: boolean;
+        contentEditable?: Booleanish;
```

Comment:
https://html.spec.whatwg.org/#dom-contenteditable

```suggestion
        contentEditable?: Booleanish | "inherit";
```

Reply:
I like it, but I don't have permission to update your fork.  Please go ahead and add it to the PR.